pkg/api/grpcServer.go

```diff
@@ -78,8 +78,10 @@ func (s *grpcServer) dispatcher(stream grpc.Stream, methodName string,
 		request proto.Message, response proto.Message)) error {
 
 	// tracks attribute state for this stream
-	tracker := s.attrMgr.NewTracker()
-	defer tracker.Done()
+	reqTracker := s.attrMgr.NewTracker()
```

Comment:
Considering we do use single trackers in tests, I like having them as singletons. They are completely independent from one another...